README.rst

```diff
@@ -1,7 +1,7 @@
 .. raw:: html
 
    <!--
-   Copyright 2015-2016 F5 Networks Inc.
+   Copyright 2015-2017 F5 Networks Inc.
```

Comment:
Does this need to change to 2016-2017? I am not sure about how copyright works for F5